legacy/src/app/partials/nodedetails/deploy-options.html

```diff
@@ -17,7 +17,7 @@
           class="u-float--left"
           style="width: auto; margin-right: 1rem"
         >
-          Register as MAAS KVM host (Ubuntu 18.04 LTS required).&nbsp;
+          Register as MAAS KVM host.&nbsp;
```

Comment:
Does this need a none breaking space?

Reply:
I guess it doesn't need it, it's just so the "Read more" link isn't so close to the checkbox label. I might actually put it in an inline list instead since that would give the standard spacing between two separate things.